canonical-web-and-design/react-components
# Prettify all files

## Done

Run all files through `yarn prettier --write src/**/*`

## QA

- Sanity check code changes
- Ensure all tests pass
- Check storybook loads as expected
